### PR TITLE
Swope errata4

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -3,11 +3,25 @@
...
+
+== Errata 2.2.5
+
+*Important:* See information in the previous notes.
+
+View a summarized list of fixes that the {product-title} Errata provides:
+
+* Fixes multiple certificate issues.
+
+* Adds labels to the Search redisgraph statefulset so that the pod is restarted automatically when certificates are refreshed. (GitHub 12299)
```
Comment (dockerymick) [inline release_notes/errata.adoc]: I recommend that you change statefulset to StatefulSet bc Kubernetes references it that way. https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/

I know that I have some inconsistencies with StatefulSet, so I will update the other instances in 2.3. Let me know your thoughts

**Review by dockerymick — APPROVED** [review]
Your changes look good. i made a comment

File: release_notes/errata.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/errata.adoc]: good idea to get them all the same and follow Kube, thanks for looking that up!

**Review by anthonyfister — APPROVED** [review]
LGTM

**Pull request merged**